libs/vm/src/parser.cpp

```diff
@@ -1440,8 +1440,8 @@ ExpressionNodePtr Parser::ParseExpressionStatement()
 bool Parser::IsStatementKeyword(Token::Kind kind) const
 {
   return (kind == Token::Kind::Persistent) || (kind == Token::Kind::Contract) ||
-         (kind == Token::Kind::EndContract) || /* (kind == Token::Kind::Struct) || */
-         /* (kind == Token::Kind::EndStruct) || */ (kind == Token::Kind::Function) ||
+         (kind == Token::Kind::EndContract) || (kind == Token::Kind::Struct) ||
```

Comment:
Not sure we want to enable this at this point do we?

Reply:
It's not enabling them though -- it's just removing one of the ways of disabling structs. Structs are actually still disabled in the parser.cpp with a commented out block.

Reply:
👍